README.md

```diff
@@ -998,3 +999,126 @@ go_proto_library(name, srcs, deps, has_services)
     </tr>
   </tbody>
 </table>
+
+### `go_embed_data`
+
+```bzl
+go_embed_data(name, src, srcs, out, package, var, flatten, string)
+```
+<table class="table table-condensed table-bordered table-params">
+  <colgroup>
+    <col class="col-param" />
+    <col class="param-description" />
+  </colgroup>
+  <thead>
+    <tr>
+      <th colspan="2">Attributes</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>name</code></td>
+      <td>
+        <code>Name, required</code>
+        <p>A unique name for the go_embed_data rule.</p>
+      </td>
+    </tr>
+    <tr>
+      <td><code>src</code></td>
```

Comment:
Why both src and srcs -- couldn't you detect a single entry in srcs and act as suggested in src?